Makefile

```
@@ -231,14 +229,12 @@ airgap-image-bundle-linux-arm64.tar \
 airgap-image-bundle-linux-arm.tar: .k0sbuild.image-bundler.stamp airgap-images.txt
 	docker run --rm -i --privileged \
 	  -e TARGET_PLATFORM='$(TARGET_PLATFORM)' \
-	  k0sbuild.image-bundler < airgap-images.txt > '$@'
+	  '$(shell cat -- '$<')' < airgap-images.txt > '$@'
 
-.k0sbuild.image-bundler.stamp: hack/image-bundler/*
-	docker build \
+.k0sbuild.image-bundler.stamp: hack/image-bundler/* embedded-bins/Makefile.variables
+	docker build --iidfile '$@' \
 	  --build-arg ALPINE_VERSION=$(alpine_patch_version) \
-	  -t k0sbuild.image-bundler \
-	  hack/image-bundler
-	touch -- '$@'
+	  -t '$(@:.%.stamp=%)' -- '$(dir $<)'
```

Trying to improve readability by being stupid and simple.

```suggestion
	  -t k0sbuild.image-bundler -- hack/image-bundler
```